.golangci.yml

```diff
@@ -10,6 +10,23 @@ linters-settings:
   misspell:
     locale: US
 
+linters:
+  enable:
+    - typecheck
+    - goimports
+    - goheader
+    - misspell
+    - govet
+    - ineffassign
+    - gosimple
+    - deadcode
+    - structcheck
+    - gomodguard
+    - gofmt
+    - unused
+    - structcheck
+    - varcheck
```

Comment:
Aren't these checkers enabled by default?

Reply:
It seems many usable linters are disabled by default. We will enable and fix respectively in different PR.

Reply:
The default linters are